vscode/src/commands/CommandsController.ts

```diff
@@ -211,15 +211,14 @@ export class CommandsController implements VsCodeCommandsController, vscode.Disp
                 case selectedCommandID === menu_options.chat.slashCommand: {
                     let input = userPrompt.trim()
                     if (input) {
-                        return await vscode.commands.executeCommand('cody.action.chat', input, {
-                            source: 'command',
-                        })
+                        await vscode.commands.executeCommand('cody.action.commands.exec', `${selectedCommandID} input`)
```

Comment:
```suggestion
-                         await vscode.commands.executeCommand('cody.action.chat', input)
```

When the selectedCommandID is equal to "/ask" , it means this is a chat question submitted from the command menu, so we can send process this as a chat question with "cody.action.chat".

Reply:
The issue with executing the command `cody.action.chat` is that we lose the context of the user selection, so the response ends up being very low quality.